Doc/library/ipaddress.rst

```diff
@@ -217,11 +217,20 @@ write code that handles both IP versions correctly.  Address objects are
       :RFC:`4291` for details.  For example,
       ``"0000:0000:0000:0000:0000:0abc:0007:0def"`` can be compressed to
       ``"::abc:7:def"``.
+
+      Optionally, the string may also have a scope (zone) id, expressed
+      with a suffix ``%<scope_id>``. If present, the scope id must be non-empty,
+      and may not contain ``%``.
+
+      See :RFC:`4007` for details.
    2. An integer that fits into 128 bits.
    3. An integer packed into a :class:`bytes` object of length 16, big-endian.
 
+
    >>> ipaddress.IPv6Address('2001:db8::1000')
    IPv6Address('2001:db8::1000')
+   >>> ipaddress.IPv6Address('ff02::5678%1')
+   IPv6Address('ff02::5678%1')
 
    .. attribute:: compressed
 
```

Comment:
I believe this documentation on `compressed` is no longer accurate. Is that right? If so, let's update it.

(Noticed this while skimming the doc just now to answer the question about where to make this feature visible with examples.)